### DISCUSSION TITLE
Summary chart

### DISCUSSION THREAD

**Discussion by Abdelahe53** [iSoron/uhabits#2034] [Feature Requests]
It would be great if you added a summary chart to all the habits and see which habits you are doing well at and the ones that's low.

**Comment by mikesiegel** [comment]
I would love the idea of a Widget that displayed an average of your scores.  I had something similar for a spreadsheet I was working on, but  Loop is much prettier. 

![life](https://github.com/user-attachments/assets/248cb8ff-725b-4917-b644-9a54a51bb33a)
